### PR TITLE
Use default settings for Brave Ads UserActivityStudy for 10% of users

### PR DESCRIPTION
**Pull request by tmancey**

### CODE REVIEW

File: seed/seed.json [hunk 1]
```diff
@@ -358,6 +358,20 @@
                         "enable_feature": ["UserActivity"]
                     }
                 },
+                {
+                    "name": "IdleTimeThreshold=5",
+                    "probability_weight": 10
```
Comment (iefremov) [inline seed/seed.json:363]: syntax err

File: seed/seed.json [hunk 2]
```diff
@@ -358,6 +358,20 @@
...
+                        {
+                            "name": "idle_time_threshold",
+                            "value": "5s"
+                        }
+                    ],
+                    "feature_association": {
+                        "enable_feature": ["UserActivity"]
+                    }
+                }
+                },
```
Comment (iefremov) [inline seed/seed.json:374]: syntax err

**Comment by iefremov** [comment]
apparently we need pr checks for this repo

File: seed/seed.json (hunk 2 above)
Comment (tmancey) [reply seed/seed.json:373]: Fixed, thanks

File: seed/seed.json (hunk 1 above)
Comment (tmancey) [reply seed/seed.json]: Fixed, thanks

**Review by iefremov — APPROVED** [review]

**Review by moritzhaller — APPROVED** [review]

**Pull request merged**